apps/analysis/views.py

```diff
@@ -0,0 +1,71 @@
+from django.shortcuts import render
+from django.db import models
+from django.utils import timezone
+
+from rest_framework.decorators import action
+from rest_framework import (
+    permissions,
+    response,
+    views,
+    viewsets,
+    serializers,
+    status
+)
+
+from deep.permissions import IsProjectMember
+
+from .models import (
+    Analysis,
+    AnalysisPillar,
+    AnalyticalStatement,
+    AnalyticalStatementEntry
+)
+from .serializers import (
+    AnalysisSerializer,
+    AnalysisPillarSerializer,
+    AnalyticalStatementSerializer,
+    AnalysisSummarySerializer,
+)
+
+
+class AnalysisViewSet(viewsets.ModelViewSet):
+    serializer_class = AnalysisSerializer
+    permission_classes = [permissions.IsAuthenticated, IsProjectMember]
+
+    def get_queryset(self):
+        return Analysis.objects.filter(project=self.kwargs['project_id']).select_related(
+            'project',
+            'team_lead',
+        )
```

Comment:
`.prefetch_related('analysispillar')` might be needed.

because of [this](https://github.com/the-deep/server/pull/667/files#diff-5a9fa181c74a2048bb546c027ac0ea8f5c2385c90535879ee315d5dc656198d2R36)